daal4py/sklearn/linear_model/_coordinate_descent.py

```diff
@@ -435,9 +435,9 @@ def _daal4py_predict_lasso(self, X):
     return res
 
 
-def _fit(self, X, y, sample_weight=None, check_input=True):
+def _fit(self, _X, _y, sample_weight=None, check_input=True):
```

Comment:
Why is this renaming required?

Reply:
https://github.com/intel/scikit-learn-intelex/pull/1848#discussion_r1630084009